src/pretix/api/auth/permission.py

```diff
@@ -59,6 +59,22 @@ def has_permission(self, request, view):
         return True
 
 
+class EventCRUDPermission(EventPermission):
+
+    def has_permission(self, request, view):
+        if not super(EventCRUDPermission, self).has_permission(request, view):
+            return False
+        elif view.action == 'create' and 'can_create_events' not in request.orgapermset:
+            return False
+        elif view.action == 'destroy' and 'can_create_events' not in request.eventpermset:
+            return False
```

Comment:
Currently, this requires ``can_change_event_settings`` permission, not ``can_create_event`` in the backend. I agree that neither choice is perfect, but we should be consistent :)

Reply:
Changed to     
```
permission = 'can_change_event_settings'
write_permission = 'can_change_event_settings'
```

Reply:
And changed back again 😄